contracts/0.8.9/WithdrawalQueueBase.sol

```diff
@@ -442,6 +442,7 @@ abstract contract WithdrawalQueueBase {
         if (_recipient == address(0)) _recipient = request.owner;
 
         request.claimed = true;
+        _getRequestsByOwner()[request.owner].remove(_requestId);
```

Comment:
maybe it worth checking the return status of the `remove` call (the same was proposed in #633)

Reply:
Probably, it is.